loopdb/store.go

```diff
@@ -550,6 +596,14 @@ func (s *boltSwapStore) CreateLoopIn(hash lntypes.Hash,
 			return err
 		}
 
+		// Store the current protocol version.
```

Comment:
I know I juust did this with labels, 😅 , but now that we have 2 fields that are common the the swap types but not in the swap contract, it may be nice to have a common function which takes the bucket and adds these fields. If we choose not to migrate swap contract, and continue with adding new keys it will be nice to have them in the same place. n/b

Reply:
I was thinking about this, and I'm unsure if that'd make the code more readable or harder to follow. Maybe I tend to be more on the "keeping them separate" side to keep them explicit rather then hide behind a function.

Reply:
indeed, was on the fence about it myself so let's leave as is.